tests/testapps/AddressBookSample/AddressBook/ViewAllContactsViewController.m

```diff
@@ -41,10 +41,30 @@ - (void)viewDidLoad {
 - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
 
+#ifdef WINOBJC
```

Comment:
Can you comment here why these ifdefs are necessary between platforms? From what I can see the split might be justified, but I'm usually suspicious of platform divergences, especially in test.